samples/Sentry.Samples.Maui/MainPage.xaml.cs

```diff
@@ -27,6 +29,28 @@ protected override void OnAppearing()
         base.OnAppearing();
     }
 
+    private void AddHandlerBreadcrumbs()
+    {
+        foreach (var visualElement in this.GetVisualTreeDescendants())
+        {
+            if (visualElement is Element element)
+            {
+                element.HandlerChanged += (sender, _) =>
+                {
+                    // The StyleId correlates to the element's name if one is set in XAML
+                    var elementName = element.StyleId;
+                    if (elementName is not null)
+                    {
+                        var type = element.Handler?.GetType();
+                        var handlerName = type is not null ? type.Name : string.Empty;
+
+                        SentrySdk.AddBreadcrumb($"'{elementName}' handler changed to '{handlerName}'", "system", "ui.handlers");
```

Comment:
I see we're accessing `SentrySdk` (static method) so not really testable (usually we'd take `IHub` ?)